lib/auth/cognito_chooser.rb

```diff
@@ -52,4 +66,13 @@ def register_stub_client
       client: CognitoStubClient.stub_client
     )
   end
+
+  def load_jwks
+    # TODO Some schedule code to reload this every n hours/days as I suspect
+    # this may change or be updated by Amazon.
+    url = "https://cognito-idp.#{region}.amazonaws.com/#{user_pool_id}/.well-known/jwks.json"
+    uri = URI(url)
+    response = Net::HTTP.get(uri)
+    SelfService.register_service(name: :jwks, client: JSON.parse(response))
```

Comment:
again, this is just a JSON, why register a whole service?

Reply:
Its used else where and is subject to change if switch between stub auth and real auth